nickytonline/remix
# fix: add type check for packages/remix-netlify-edge

Adds type checking for packages/remix-netlify-edge

Relates to #3104


Testing Strategy:

1. Clone this repository
2. Checkout the branch, e.g. `gh co 2`
3. Install all dependencies by running `yarn`
4. Run `node scripts/build.mjs --deno` from the root folder of the project in your shell.
5. All type checks should pass.
